libmscore/edit.cpp

```diff
@@ -1810,7 +1811,7 @@ void Score::deleteItem(Element* el)
         undoAddCR(rest, segment->measure(), segment->tick());
     }
     // fall through
-
+    case ElementType::MMREST:
```

Comment:
Interesting thought - can there ever be an mmrest on voice > 0?  Would that ever made sense for any reason?